docs/source/sdk.rst

```diff
@@ -10,12 +10,23 @@ An SDK is a JS library that helps to communicate with a relayer. The SDK makes i
 Creating an SDK
 ---------------
 
-**new UniversalLoginSDK(relayerURL, providerURL, messageOptions)**
+**new UniversalLoginSDK(relayerURL, providerURL, sdkConfig)**
 
   Parameters:
     - **relayerURL** : string - a URL address of a relayer
     - **providerURL** : string - JSON-RPC URL of an Ethereum node
-    - **messageOptions** (optional) : object - specific message options as ``gasPrice`` or ``gasLimit``
+    - **sdkConfig** (optional) : object - specific sdk options listed below:
+
+      - applicationInfo : object - detailed info about application: applicationName, type (e.g. laptop, mobile), logo (image source)
+      - paymentOptions : object - specific transaction options: gasLimit, gasPrice, gasToken
```

Comment:
We will also need an example here

Reply:
I will add example for the whole sdkConfig